src/crop_image.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+from os import listdir
+from os.path import isfile, join, splitext
+from PIL import Image
+import numpy as np
+import cv2
+import glob
+
+
+def crop(image_path, coords, saved_location):
+    """
+    @param image_path: The path to the image to edit
```

Comment:
Its best if function headers should have a description, the parameters, and the return value, see https://github.com/julianweisbord/Deep-Learning-For-Object-Recognition-on-a-Mobile-Robot/blob/master/src/learning/model.py